includes/data-port/class-sensei-data-port-manager.php

```diff
@@ -0,0 +1,201 @@
+<?php
+/**
+ * File containing the Sensei_Data_Port_Manager class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * This class is responsible creating, deleting and keeping track of data port jobs.
+ */
+class Sensei_Data_Port_Manager implements JsonSerializable {
```

Comment:
Similar comment to another PR regarding the "Port" word. Perhaps should replace it with "Import_Export"?

Reply:
Have a look at my answer [here](https://github.com/Automattic/sensei/pull/3099#discussion_r422894254).